pages/linux/nmblookup.md

```diff
@@ -0,0 +1,11 @@
+# nmblookup
+
+> Discover SMB shares.
+
```

Comment:
```suggestion
> More information: <https://www.samba.org/samba/docs/current/man-html/nmblookup.1.html>.

```
Thanks for your contribution and welcome to tldr. I would suggest adding a link to the docs/website. 
I am not a user of this command so I will leave the review to others. Can you also sign the CLA?